### DISCUSSION TITLE
FlareSolverr Support

### DISCUSSION THREAD

**Discussion by stranger-danger-zamu** [lncrawl/lightnovel-crawler#1283] [Ideas]
[FlareSolverr](https://github.com/FlareSolverr/FlareSolverr) is a API that wraps around a headless browser and puppeteer which bypasses a lot of the issues people have with CloudFlare.

Having a CLI option or a configuration setting which allows uses to use a flaresolverr instance would be very helpful.

**Comment by dipu-bd** [comment]
Good idea. I was also thinking of adding support of selenium via docker. FlareSolverr looks more promising. It will take some work to make a client to connect with it. I will check it on my free time (which is a bit rare these days 😔).
